config.yaml

```diff
@@ -0,0 +1,420 @@
+              sizeGb: 60
+              sourceImage: projects/centos-cloud/global/images/family/centos-7
+              type: project/zones/us-central1-a/diskTypes/pd-ssd        
+          machineType: zones/us-central1-b/machineTypes/n1-highcpu-4
+          disks[].autoDelete: False
+          
+    - type: gce_instance_v2.py
+      name: NFS 
+      properties:
+          - network: *NETWORK 
+            subnetworks: *NETWORK_PROD_DATA
+        disks:
+            - deviceName: 'boot'
+              boot: true 
+              sizeGb: 60
+              sourceImage: projects/centos-cloud/global/images/family/centos-7
+              type: project/zones/us-central1-b/diskTypes/pd-ssd        
+          machineType: zones/us-central1-b/machineTypes/n1-highcpu-4
+          disks[].autoDelete: False
+          
+# Instances un-managed:
+    - type: compute.v1.instance
+      name: Magento_Admin_boot
+      properties:
+          disks:
+            - name: 'boot'
+              boot: true
+              sizeGb: 60
+              sourceImage: projects/centos-cloud/global/images/family/centos-7
+              type: project/zones/us-central1-a/diskTypes/pd-ssd             
+          machineType: zones/us-central1-a/machineTypes/n1-standard-4
+          tag: *NETWORK_PROD_WEB
+          disks[].autoDelete: False
+          
+    - type: compute.v1.instance
+      name: Magento_Admin_Data_1
+      properties:
+          disk:
+            - deviceName: 'Data_1'
+              sizeGb: 100
+              sourceImage: projects/centos-cloud/global/images/family/centos-7
+              type: project/zones/us-central1-a/diskTypes/pd-ssd             
+          machineType: zones/us-central1-a/machineTypes/n1-standard-4
+          tag: *NETWORK_PROD_WEB
+          disks[].autoDelete: False          
+
+    - type: compute.v1.instance
+      name: Magento_Admin_Data_2
+      properties:
+          disk:
+            - deviceName: 'Data_2'
+              sizeGb: 100
+              sourceImage: projects/centos-cloud/global/images/family/centos-7
+              type: project/zones/us-central1-a/diskTypes/pd-ssd             
+          machineType: zones/us-central1-a/machineTypes/n1-standard-4
+          tag: *NETWORK_PROD_WEB
+          disks[].autoDelete: False
+
+
+# Storage bucket ######################################
+    - type: bucket_v1.py
+      name: Backup
+      properties:
+         location: US
+         storageClass: MULTI_REGIONAL
+
+# Cloud SQL ############################################
+
+    - type: cloudsql_instance_v1.py
+      name: Production 
+      properties:
+         region: us-central
+         settings:
+           tier: db-n1-standard-2
+           dataDiskType: PD_HDD
+           dataDiskSizeGb: 250
+           locationPreference:
+             zone: us-central1-a
+         databaseVersion: MYSQL_5_6
+         backendType: SECOND_GEN  
+         instanceType: CLOUD_SQL_INSTANCE
+
+  - type: cloudsql_instance_v1.py
```

Comment:
If we are creating a read replica, consider using a failover read replica.

Reply:
Need to check Docs.